src/renderable/animationsheet.js

```diff
@@ -19,11 +19,12 @@
      * @param {Number} [settings.framewidth] Width of a single frame within the spritesheet
      * @param {Number} [settings.frameheight] Height of a single frame within the spritesheet
```

Comment:
I'm talking about the Sprite's anchorPoint, not the Entity's (parent's) anchorPoint. The Sprite's left property would be adjusted based off the Sprite's anchorPoint.

![spriteanchor](https://cloud.githubusercontent.com/assets/4075314/8997064/63b6d8ea-36eb-11e5-8ca4-4c34e81a85ce.png)

Anyhow, when I modify the left and top to adjust based on the anchor point if it exists, a jasmine test for the droptarget entity fails. It seems the top,left,bottom,right values should be adjusted *only* for Sprites, not all Renderables.

Making up for this in the debug panel seems hacky to me.